pages/api/file/upload.ts

```diff
@@ -17,10 +17,23 @@ export default async function handle(req: NextApiRequest, res: NextApiResponse):
     }
 
     // TODO make this more robost/better
-    const accepted_type_paths = ["criminal-check", "income-proof", "curriculum-plans"];
+    const accepted_type_paths = [
+        "criminal-check",
+        "income-proof",
+        "curriculum-plans",
+        "cover-photo",
+    ];
 
     const { path, file } = req.query;
 
+    let bucket = "";
+
+    if (path === "cover-photo") {
+        bucket = process.env.S3_PUBLIC_IMAGES_BUCKET;
```

Comment:
In the case where we are uploading a cover-photo, we do not want any user specific subfolders. We want to put the file under the `{program,class}/[id]` path instead.